package.json

```diff
@@ -38,13 +38,13 @@
     }
   },
   "devDependencies": {
-    "@vitest/coverage-c8": "0.33.0",
+    "@vitest/coverage-v8": "1.1.0",
     "ajv": "8.12.0",
     "ajv-formats": "2.1.1",
     "coveralls": "3.1.1",
     "prettier": "3.1.1",
     "typescript": "5.3.3",
-    "vitest": "0.34.6"
```

Comment:
I had to do this Vitest upgrade to successfully run the tests locally. I'm happy to remove it if you have another upgrade process, but I don't think it should cause any issues.